### PR TITLE
krfb: expose kde 5 version

### PR DESCRIPTION
**Pull request by jerith666**
###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/kde/krfb.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  extra-cmake-modules, kdoctools,
+  kdelibs4support, kdnssd, libvncserver, libXtst
+}:
+
+let
+  unwrapped =
+    kdeApp {
+      name = "krfb";
+      meta = {
+        license = with lib.licenses; [ gpl2 fdl12 ];
```
Comment (ttuegel) [inline pkgs/applications/kde/krfb.nix]: Please add yourself to `meta.maintainers`!

**Review by ttuegel — CHANGES_REQUESTED** [review]
Please rename the commit to `krfb: init at 16.12.2` to conform to our style guide and add yourself to the maintainers. Looks good otherwise!

**Comment by jerith666** [comment]
Heh, I had originally written the log as "init at ...", but then I figured that wasn't right since it had already been present in the kde 4 tree.  Ah well!  :)  Both changes made.

**Review by ttuegel — APPROVED** [review]

**Comment by ttuegel** [comment]
> Heh, I had originally written the log as "init at ...", but then I figured that wasn't right since it had already been present in the kde 4 tree. Ah well! :) Both changes made.

Well, it's tricky, because I already removed the KDE 4 version! But I see your point.

**Pull request merged**